### PR TITLE
Migrate to datasketches 2.2.0b1

### PR DESCRIPTION
**Pull request by andyndang**
* Fixed cross language serialization issue
* Add tests for the latest Java serialization

### CODE REVIEW

File: src/whylogs/core/statistics/numbertracker.py [hunk 1]
```diff
@@ -153,10 +157,10 @@ def from_protobuf(message: NumbersMessage):
         number_tracker : NumberTracker
         """
         theta = None
-        if message.theta is not None and len(message.theta) > 0:
-            theta = ThetaSketch.deserialize(message.theta)
-        elif message.compact_theta is not None and len(message.compact_theta) > 0:
+        if message.compact_theta is not None and len(message.compact_theta) > 0:
```
Comment (lalmei) [inline src/whylogs/core/statistics/numbertracker.py:160]: why did the order change ?

**Review by lalmei — APPROVED** [review]
LGTM, just one question.

File: src/whylogs/core/statistics/numbertracker.py (hunk 1 above)
Comment (andyndang) [reply src/whylogs/core/statistics/numbertracker.py:160]: `.theta` is no longer supported by datasketches. Only compact serialization

Comment (lalmei) [reply src/whylogs/core/statistics/numbertracker.py:160]: 👍

**Pull request merged**